chart/org.eclipse.birt.chart.ui.extension/src/org/eclipse/birt/chart/ui/swt/composites/ImageDialog.java

```diff
@@ -104,9 +106,13 @@ public class ImageDialog extends TrayDialog {
 	protected SelectType selectType;
 
 	/**
-	 * The constructor.
+	 * The constructor 1
```

Comment:
This looks like a type.
```suggestion
	 * The constructor.
```